### PR TITLE
[aws-load-balancer-controller] allow --aws-region override

### PR DESCRIPTION
**Pull request by dza89**
Issue #, if available:

Description of changes:

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: stable/aws-load-balancer-controller/README.md [hunk 1]
```diff
@@ -142,4 +142,5 @@ The following tables lists the configurable parameters of the chart and their de
 | `serviceAccount.name`              | Service account to be used                                | None                                                                                       |
 | `terminationGracePeriodSeconds`    | Time period for controller pod to do a graceful shutdown  | 10                                                                                         |
 | `ingressClass`                     | The ingress class to satisfy                              | alb                                                                                        |
+| `region`                          | Set to use KIAM or kube2iam for example                             |
```
Comment (kishorj) [inline stable/aws-load-balancer-controller/README.md:145]: Lets say something like this in the description, -
The AWS region for the kubernetes cluster.

File: stable/aws-load-balancer-controller/values.yaml [hunk 2]
```diff
@@ -72,6 +72,9 @@ enableCertManager: false
 # ingresses without ingress class annotation and ingresses of type alb
 ingressClass: alb
 
+# Set to use KIAM or kube2iam for example
```
Comment (kishorj) [inline stable/aws-load-balancer-controller/values.yaml:75]: Lets say something like
The AWS region for the kubernetes cluster. Set to use KIAM or kube2iam for example.

**Review by kishorj — COMMENTED** [review]
Thank you for the pull request. I've some minor comments on the documentation.

**Comment by dza89** [comment]
@kishorj  updated per your request

**Review by kishorj — APPROVED** [review]

**Review by jaypipes — APPROVED** [review]
Changed the title of the PR since it's not actually specific to kiam...

**Comment by dza89** [comment]
@jaypipes @kishorj 

How are new charts published? If i do a helm pull eks/aws-load-balancer-controller i'm still pulling 1.0.0

**Comment by kishorj** [comment]
> @jaypipes @kishorj
> 
> How are new charts published? If i do a helm pull eks/aws-load-balancer-controller i'm still pulling 1.0.0

We release new charts periodically - whenever there are significant changes or critical fixes. New chart for eks/aws-load-balancer-controller hasn't been released since 1.0.0. We plan to release one this week.

**Comment by jellis** [comment]
Conceptually, with this running in Fargate, should we also be allowing the creation of an argument to define vpc-id?

See here https://github.com/kubernetes-sigs/aws-load-balancer-controller/blob/efe7e71d270e57a70d0c17587ee89237e055ee4d/pkg/aws/cloud.go#L44

**Comment by jellis** [comment]
> Conceptually, with this running in Fargate, should we also be allowing the creation of an argument to define vpc-id?
> 
> See here https://github.com/kubernetes-sigs/aws-load-balancer-controller/blob/efe7e71d270e57a70d0c17587ee89237e055ee4d/pkg/aws/cloud.go#L44

Ignore that comment - I see a PR has just gone in for it https://github.com/aws/eks-charts/pull/307

**Pull request merged**